_includes/getting-started.html

```diff
@@ -9,9 +9,9 @@
     <div id="getting-started">
     <div style="margin-top: auto;" id="box"> 
       <h1 style="text-align: left;margin-left: 80px;"><strong style="font-size: 90px;">Get Started</strong><br>
-        with <span>Meshery</span> in<br>
+        with Meshery in<br>
```

Comment:
No need to remove the span here

Reply:
It's an errant tag, though, right? Residual from your removal of the blue highlight. I wonder if there's value in leaving it.